src/pages/home/components/feature-description/feature-explanation-desktop.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from "react"
+import { spaces, textStyles } from "../../../../styles"
+import styled from "styled-components"
+import Anchor from "../../../../components/anchor"
+
+
+const Container = styled.div`
+  background-image: url(${props => props.image});
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  height: 400px;
+  margin-top: 24px;
+  width: 100%;
+`
+const Headline = styled.h2`
+  margin-right: 120px;
+  padding-top: 100px;
+  text-align: right;
+  ${textStyles.h2Bold}
+`
+
+const Checklist = styled.img`
+  align-self: center;
+  height: ${spaces.relax}px;
+  margin-right: ${spaces.cozy}px;
+  margin-top: 0;
+  width: ${spaces.relax}px;
+`
+
```

Comment:
make sure you fix all the comments from `feature-explanation-mobile.js` in here too.